### ISSUE TITLE
Locator ID Not Correct

### ISSUE THREAD

**Issue by jason-bean** [google/sagetv#245]
My locator ID is showing as xxxx-xxxx-0000-0000 where the last two sequences are just all zeros. I know my server responds with a different locator ID but I'm not sure it is still working. I upgraded my router a couple weeks ago and using the locator ID no longer works. There seems to be something wrong with this code.

**Comment by jason-bean** [comment]
BTW, I'm using SageTV through the Docker image on unRAID.

I had recorded the locator ID that was originally generated. I stopped SageTV and modified the Sage.properties file so that locator/id is the original generated. Then started it back up. On startup the locator ID went back to where the last two sequences are all zeros.

**Comment by enternoescape** [comment]
Do you see a line that looks like this in your log:
`MAC address was not able to be detected, using zero for the MAC!`

**Comment by jason-bean** [comment]
No, I'm not seeing that in my log but that seems to be what happened.

**Comment by Narflex** [comment]
He's right that the MAC being read as all zeros would cause this.

Jeff Kardatzke
Sent from my Android

On Jan 10, 2017 9:30 AM, "Jason Bean" <notifications@github.com> wrote:

No, I'm not seeing that in my log but that seems to be what happened.

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/google/sagetv/issues/245#issuecomment-271640881>, or mute
the thread

**Comment by jason-bean** [comment]
Interestingly I had tried connecting with the zeros earlier and Placeshifter reported that the servers wasn't found. Now it seems to be working fine with the zeros.

This should probably still be looked at. I should note that if this is unrelated to unRAID and Docker that my system has dual network adapters if that would cause confusion with SageTV obtaining a MAC.

Now that I'm thinking about it I believe when I originally installed SageTV on my unRAID system I had the two NIC's bonded. For noise reasons I had to take the switch out that I was using so currently just using one of the NIC's. The other is not connected.
